lib/galaxy/schema/drs/gen.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# must be run from a virtualenv with...
+# https://github.com/koxudaxi/datamodel-code-generator
+for model in AccessMethod Checksum DrsObject Error AccessURL ContentsObject DrsService
+do
+    datamodel-codegen --url "https://raw.githubusercontent.com/ga4gh/data-repository-service-schemas/master/openapi/components/schemas/${model}.yaml" --output "$model.py"
```

Comment:
Can you use a permalink, e.g a tag or a commit in urls ?

Reply:
I mean ... we wouldn't want to regenerate this if nothing changed 😆.